src/components/onboarding/email/enter-email-component.vue

```diff
@@ -0,0 +1,38 @@
+<style lang="scss" scoped>
+@import './enter-email-component.scss';
+</style>
+
+<template>
+  <div class="intro--enter-email">
+    <h1>Track Progress</h1>
+    <p>Enter your email to track your progress and be rewarded!</p>
+    <input :class="{'valid': validEmail}" type="text" name="email" id="email" placeholder="ILove@DenStartupWeek.com" v-model="entered" >
```

Comment:
Could we go ahead and make the placeholder a variable here?  You can default it to `ILove@DenStartupWeek.com`, I just don't want to forget about it.  😂